test/jdk/javax/crypto/CryptoPermissions/InconsistentEntries.java

```diff
@@ -23,57 +23,87 @@
  * questions.
  */
 
-/**
+/*
  * @test
  * @bug 8286779
  * @summary Test limited/default_local.policy containing inconsistent entries
- * @run main/manual InconsistentEntries
+ * @library /test/lib
+ * @run driver InconsistentEntries
```

Comment:
Use the testng framework.

Reply:
Same reason as Root.java. The custom policy file needs to place in Java home directory before running the test.